tutormfe/plugin.py

```diff
@@ -15,34 +16,23 @@
         "CADDY_DOCKER_IMAGE": "{{ DOCKER_IMAGE_CADDY }}",
         "ACCOUNT_MFE_APP": {
             "name": "account",
-            "repository": "https://github.com/edx/frontend-app-account",
+            "repository": "https://github.com/openedx/frontend-app-account",
             "port": 1997,
-            "env": {
-                "production": {
-                    "COACHING_ENABLED": "",
-                    "ENABLE_DEMOGRAPHICS_COLLECTION": "",
-                },
-            },
```

Comment:
Doesn't this still need to be added to LMS settings using `MFE_CONFIG_OVERRIDES`?

Reply:
We can but those values are already empty strings, so I though they are not necessary

Reply:
You are right: https://github.com/openedx/frontend-app-account/blob/master/.env . Nevermind then, your change looks good.

I wonder why they here in the first place.